sdk/scheduler/src/test/java/com/mesosphere/sdk/dcos/CapabilitiesTest.java

```diff
@@ -1,148 +1,88 @@
-        mockDcosCluster = MockDcosCluster.create("1.7.0");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertFalse(capabilities.supportsRLimits());
+    public void test_1100() throws IOException {
+        testClusterVersion("1.10.0", true, true);
     }
 
     @Test
-    public void testHasRLimitsFails_1_7dev() throws URISyntaxException, IOException {
-        mockDcosCluster = MockDcosCluster.create("1.7-dev");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertFalse(capabilities.supportsRLimits());
+    public void test_110dev() throws IOException {
+        testClusterVersion("1.10-dev", true, true);
     }
 
     @Test
-    public void testHasRLimitsFails_1_8() throws URISyntaxException, IOException {
-        mockDcosCluster = MockDcosCluster.create("1.8.0");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertFalse(capabilities.supportsRLimits());
+    public void test_200() throws IOException {
+        testClusterVersion("2.0.0", true, true);
     }
 
     @Test
-    public void testHasRLimitsFails_1_8dev() throws URISyntaxException, IOException {
-        mockDcosCluster = MockDcosCluster.create("1.8-dev");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertFalse(capabilities.supportsRLimits());
+    public void test_20dev() throws IOException {
+        testClusterVersion("2.0-dev", true, true);
     }
 
-    @Test
-    public void testHasRLimitsSucceeds_1_9() throws URISyntaxException, IOException {
-        mockDcosCluster = MockDcosCluster.create("1.9.0");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertTrue(capabilities.supportsRLimits());
-    }
-
-    @Test
-    public void testHasRLimitsSucceeds_1_9dev() throws URISyntaxException, IOException {
-        mockDcosCluster = MockDcosCluster.create("1.9-dev");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertTrue(capabilities.supportsRLimits());
-    }
-
-    @Test
-    public void testHasRLimitsSucceeds_2_0() throws URISyntaxException, IOException {
-        mockDcosCluster = MockDcosCluster.create("2.0.0");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertTrue(capabilities.supportsRLimits());
-    }
-
-    @Test
-    public void testHasRLimitsSucceeds_2_0dev() throws URISyntaxException, IOException {
-        mockDcosCluster =  MockDcosCluster.create("2.0-dev");
-        Capabilities capabilities = new Capabilities(mockDcosCluster.getDcosCluster());
-        Assert.assertTrue(capabilities.supportsRLimits());
+    private void testClusterVersion(String version, boolean expectNamedVips, boolean expectRLimits) throws IOException {
```

Comment:
I think this may be a case where keeping things DRY will make adding test cases more work than necessary down the line -- maybe keep the mock setup in this function, but perform the asserts on a per-test basis? That way, if there's some new capability we need to add tests for in the future, we don't have to modify this function signature as well as each existing test in this module. I think some redundancy in tests makes them more readable, anyway -- I want to be able to read a test and see plainly what behavior is being asserted as much as possible, instead of having to chase function invocations around.